projects/observability/src/shared/components/gauge-list/gauge-list.component.ts

```diff
@@ -71,18 +74,28 @@ export class GaugeListComponent<T extends GaugeItem = GaugeItem> implements OnCh
       return;
     }
 
-    let maxValue = maxBy(this.items, option => option.value)?.value;
-    if (maxValue === undefined || maxValue === 0) {
-      maxValue = 1;
+    if (this.total !== undefined && this.total !== 0) {
+      this.setItemOptions(this.items, this.total, this.total);
```

Comment:
There's two different values here - totalCount looks like it got added later (not sure I agree with that from a visualization user case perspective, but a bit besides the point), and it represents that total that would be passed in here. FullCount is the visual indicator, representing the max value.

So I think there's a decent argument to be made to fix those percentages by passing the accurate total from the parent, but the bars should still be based on max value. Once we change that, we're really changing the whole purpose of the visualization.